src/Microsoft.Data.SqlClient/tests/FunctionalTests/AADAuthenticationTests.cs

```diff
@@ -50,6 +50,24 @@ private void InvalidCombinationCheck(SqlCredential credential)
             }
         }
 
+        /// <summary>
+        /// Tests whether SQL Auth provider is overridden using app.config file.
+        /// This use case is only supported for .NET Framework applications, as driver doesn't support reading configuration from appsettings.json file.
+        /// In future if need be, appsettings.json support can be added.
+        /// </summary>
+        /// <returns></returns>
+        [ConditionalFact(typeof(TestUtility), nameof(TestUtility.IsFullFramework))]
+        public async Task IsDummySqlAuthenticationProviderSetByDefault()
```

Comment:
This test seems to run fine locally but I haven't been able to run it in pipelines for some reason. I would disable it in pipelines for now, with a comment to address it in sometime future to unblock releasing the fix.

Reply:
No-op. Resolved.